client/app/scripts/liveblog-edit/views/draft-posts-list.html

```diff
@@ -1,7 +1,7 @@
 <div class="draft-posts">
     <ul>
         <lb-post
-          ng-repeat="post in draftPosts.posts track by post._id"
+          ng-repeat="post in draftPosts.posts | orderBy: '-_created' track by post._id"
```

Comment:
That will give us the last updated post on top of post list, and we don't want that.